docker-compose.yml

```diff
@@ -31,9 +31,10 @@ services:
       EQ_ENABLE_CACHE: "True"
       EQ_DEVELOPER_LOGGING: "True"
       EQ_RABBITMQ_ENABLED: "True"
-      EQ_SECRET_KEY: "NOT_SECRET_DOCKER_KEY"
-      EQ_SERVER_SIDE_STORAGE_DATABASE_URL: postgres://digitaleq:digitaleq@db:5432/runner
-      EQ_RABBITMQ_URL: amqp://digitaleq:digitaleq@rabbit:5672/%2F
+      EQ_SECRETS_FILE: ".dev_secrets.yml"
+      EQ_SERVER_SIDE_STORAGE_DATABASE_URL: digitaleq:digitaleq@db:5432/runner
+      EQ_RABBITMQ_URL: digitaleq:digitaleq@rabbit:5672
+      EQ_RABBITMQ_URL_SECONDARY: digitaleq:digitaleq@rabbit:5672
```

Comment:
I think these need to be replaced with the new host and port environment variables